types/react-native/v0.65/index.d.ts

```diff
@@ -9489,7 +9489,8 @@ export interface KeyboardEvent extends Partial<KeyboardEventIOS> {
 
 type KeyboardEventListener = (event: KeyboardEvent) => void;
 
-export interface KeyboardStatic extends NativeEventEmitter {
+export interface KeyboardStatic {
+    _emitter: NativeEventEmitter
```

Comment:
Should we add those properties? I assume the `_` marks them as private?

Reply:
I don't know. It definitely makes my life easier if it is there, but you're right that it was likely intended as private. I'm not sure how the flow types handle private fields.

Reply:
I just looked it up as I wasn’t aware of it myself. Seems like underscores can be configured as private properties (https://flow.org/en/docs/config/options/#toc-munge-underscores-boolean) and the react-native repo seems to use it (https://github.com/facebook/react-native/blob/main/.flowconfig#L40).

I personally would remove it from the public typings and not expose it in the types...

Reply:
Just because something exists in source code doesn't mean it should be exposed via types.

Do you have links to flowtypes or documenation mentioning this property?

Reply:
Yeah, I can go ahead and remove it and just override it locally 👍